akka-distributed-data/src/main/scala/akka/cluster/ddata/ORSet.scala

```diff
@@ -392,15 +391,15 @@ final class ORSet[A] private[akka] (
    * This has the same result as using [[#remove]] for each
    * element, but it is more efficient.
    */
-  def clear(node: SelfUniqueAddress): ORSet[A] = clear(node.uniqueAddress)
+  def clear(@unused node: SelfUniqueAddress): ORSet[A] = clear()
 
   @deprecated("Use `remove` that takes a `SelfUniqueAddress` parameter instead.", since = "2.5.20")
-  def clear(node: Cluster): ORSet[A] = clear(node.selfUniqueAddress)
+  def clear(@unused node: Cluster): ORSet[A] = clear()
 
   /**
    * INTERNAL API
    */
-  @InternalApi private[akka] def clear(node: UniqueAddress): ORSet[A] = {
+  @InternalApi private[akka] def clear(): ORSet[A] = {
```

Comment:
What wat, it wasn't the `node` actually needed? Shouldn't that be used to create a new version vector like in `remove`?

Reply:
That does seem counter-intuitive. On the other hand, the informal definition of the algorithm does say the version vector is only updated only on additions, not removals. At the remove, it is true a new VersionVector is created to use in the delta, but it doesn't seem to be incremented there either. So this might be correct but I can't say I fully digested the algorithm yet ;) /cc @patriknw

Reply:
In public api it's included to be consistent with the other operations and then I guess it just followed along down here to the impl method although not needed. Should be ok to remove.

Reply:
Thanks for clarifying P